ts/pages/governance/register.tsx

```diff
@@ -0,0 +1,235 @@
+import { Splitview } from 'ts/components/staking/wizard/splitview';
+import { StartStaking, VotingPowerConfirmation } from 'ts/components/staking/wizard/wizard_flow';
+import { ConfirmationWizardInfo, VotingPowerWizardInfo } from 'ts/components/staking/wizard/wizard_info';
+import { Heading } from 'ts/components/text';
+import { Text } from 'ts/components/ui/text';
+import { useAPIClient } from 'ts/hooks/use_api_client';
+import { RegisterRouterSteps, useRegisterWizard } from 'ts/hooks/use_register_wizard';
+import { useStake } from 'ts/hooks/use_stake';
+import { State } from 'ts/redux/reducer';
+import { colors } from 'ts/style/colors';
+import { AccountReady, Epoch, PoolWithStats, ProviderState, StakingPoolRecomendation, WebsitePaths } from 'ts/types';
+import { DEFAULT_POOL_ID } from 'ts/utils/configs';
+import { constants } from 'ts/utils/constants';
+import { formatZrx } from 'ts/utils/format_number';
+
+interface IRegisterWizardProps {
+  providerState: ProviderState;
+  onOpenConnectWalletDialog: () => void;
+}
+
+const Container = styled.div`
+    max-width: 1390px;
+    margin: 0 auto;
+    position: relative;
+`;
+
+export const RegisterWizard: React.FC<IRegisterWizardProps> = props => {
+  const { providerState, onOpenConnectWalletDialog } = props;
+  const networkId = useSelector((state: State) => state.networkId);
+  const apiClient = useAPIClient(networkId);
+  const { currentStep, next } = useRegisterWizard();
+  const [nextEpochStart, setNextEpochStart] = React.useState<Date | undefined>(undefined);
+  const [stakingPools, setStakingPools] = React.useState<PoolWithStats[]>(undefined);
+  const [selectedPool, setSelectedPool] = React.useState<StakingPoolRecomendation>({ pool: undefined, zrxAmount: undefined});
+  const [nextEpochStats, setNextEpochStats] = React.useState<Epoch | undefined>(undefined);
+  const stake = useStake(networkId, providerState);
+  const [shouldShowDecisionScreen, setShowDecisionScreen] = React.useState<boolean>(true);
+
+  const { zrxBalanceBaseUnitAmount, address } = providerState.account as AccountReady;
+  let zrxBalance: BigNumber | undefined;
+  if (zrxBalanceBaseUnitAmount) {
+      zrxBalance = Web3Wrapper.toUnitAmount(zrxBalanceBaseUnitAmount, constants.DECIMAL_PLACES_ZRX);
+  }
+
+  let roundedZrxBalance;
+  if (zrxBalance) {
+      roundedZrxBalance = formatZrx(zrxBalance).roundedValue;
+  }
+
+  React.useEffect(() => {
+    // tslint:disable-next-line: no-floating-promises
+    (async () => {
+      const [epochsResponse, poolsResponse] = await Promise.all([
+        apiClient.getStakingEpochsWithFeesAsync(),
+        apiClient.getStakingPoolsAsync(),
+      ]);
+
+      const epochStart = epochsResponse.nextEpoch && new Date(epochsResponse.nextEpoch.epochStart.timestamp);
+      setNextEpochStats(epochsResponse.nextEpoch);
+      setNextEpochStart(epochStart);
+      setStakingPools(poolsResponse.stakingPools);
+    })();
+  }, []);
```

Comment:
i think this might be missing `apiClient` in the dep array

Reply:
Added the missing dependencies